integration/binauthz_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2023 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/v2/integration/skaffold"
+)
+
+const (
+	binauthzClusterName = "integration-tests-binauthz"
+)
+
+// TestBinAuthZWithDeploy targets the integration-tests-binauthz cluster on k8s-skaffold which has '--binauthz-evaluation-mode=PROJECT_SINGLETON_POLICY_ENFORCE' set
+func TestBinAuthZWithDeploy(t *testing.T) {
+	MarkIntegrationTest(t, NeedsGcp)
+	if os.Getenv("GKE_CLUSTER_NAME") != binauthzClusterName {
```

Comment:
I think we need to do a change in Kokoro to make this work, right?

Reply:
Yes you're correct - currently this won't run in kokoro (but won't affect the test negatively).  Plan is to make the kokoro change to add this cluster and possible modify the test harness as well to only run this test in that cluster